scona/scripts/visualisation_commands.py

```diff
@@ -54,22 +54,47 @@ def rescale(fname, suff='png'):
     # And you're done!
 
 
-def view_corr_mat(corr_mat_file,
+def view_corr_mat(corr_mat,
                   output_name,
                   cmap_name='RdBu_r',
                   cost=None,
                   bin=False):
-    ''' This is a visualisation tool for correlation matrices'''
+    '''
+    This is a visualisation tool for correlation matrices
+
+    Parameters
+    ----------
+    corr_mat : :class:`pandas.DataFrame` or :class:`str`
```

Comment:
This logic makes sense to me! Thanks for clearly explaining your thought process in the PR 😸